src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindowBase.cs

```diff
@@ -195,12 +203,12 @@ private void SpatialLocalizationGUI(string deviceTypeLabel, SpatialCoordinateSys
             GUILayout.BeginHorizontal();
             {
                 selectedLocalizerIndex = EditorGUILayout.Popup(selectedLocalizerIndex, localizerNames);
-                if (localizers != null)
+                if (localizers != null &&
+                    localizers.Length> selectedLocalizerIndex)

```

Comment:
>localizers.Length> selectedLocalizerIndex [](start = 20, length = 41)

Nit: I know it's logically equivalent but I would usually think of this as "selectedLocalizerIndex < localizers.Length"